pkgs/by-name/tf/tf/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

Let replace the `rec` construct by a `finalAttrs` as this also more flexibility, notably with overrides:
```suggestion
buildGoModule (finalAttrs: {
```
- #315337